test/test-build.js

```diff
@@ -39,6 +39,18 @@ function doTests(transpiler) {
     });
   });
 
+  test('Static string build', function () {
+    builder.reset();
+    builder.config({ transpiler: transpiler });
+    return builder.bundle('foo.js', {
+      fetch: function (data) {
+        if (data.name.indexOf('foo.js') !== -1) {
+          return fs.readFileSync('test/fixtures/test-tree/amd-1.js', 'utf8');
+        }
+      }
```

Comment:
Nice API! How about injecting this into the loader itself by implementing in the `lib/builder.js` reset() function - `var loaderFetch = builder.loader.fetch; builder.loader.fetch = function(load) { return (builder.fetch || loaderFetch).call(this, load); }` and then having this config option set `builder.fetch`?